docs/references/message-execution-properties.mdx

```diff
@@ -51,12 +51,18 @@ Property 3 provides a guarantee on the execution order of messages on a target c
 
 ![example_orderings](./_attachments/example_orderings.png)
```

Comment:
Property 4 also talks a lot about "message [executions]". It should also be updated.

Reply:
Done